R/models.R

```diff
@@ -302,10 +310,27 @@ features.ranger <- function(x, feature_labels) {
 
 #' @rdname features
 features.gam <- function(x, feature_labels) {
+
+  if(!is.null(feature_labels)) feautures_message()
+
   tail(all.vars(x$terms), -1)
 }
 
 #' @rdname features
 features.xgb.Booster <- function(x, feature_labels) {
+
+  if(!is.null(feature_labels)) feautures_message()
+
   x$feature_names
 }
+
+#' @keywords internal
+feautures_message <- function(x) {
```

Comment:
Perfect, except typo in function name :-)

Reply:
Fixed with ea81e5e.